nvidia/packages/index.rst

```diff
@@ -11,8 +11,9 @@ Show the usage of application in NVIDIA platform.
    :maxdepth: 1
 
    jetpack_sdk/index.rst
+   driver_packages/index.rst
```

Comment:
I think this should be put under Advanced Usage.

Reply:
OK